README.md

```diff
@@ -28,37 +28,56 @@ A very basic Sculpin based blog supporting the following features:
 Install
 -------
 
+This application uses [Symfony's Webpack Encore](https://symfony.com/doc/current/frontend.html)
+to manage CSS, JavaScript and image assets. First install the JS dependencies:
+
+```bash
+$ yarn install
+```
+
+Then install the PHP dependencies (Sculpin):
+
 ```bash
 $ composer install
 ```
 
 Build
 -----
 
-    php vendor/bin/sculpin generate --watch --server
+First run Encore, to compile the assets in `source/assets/` into `source/build/`:
+
+```bash
+$ yarn encore dev
```

Comment:
would yarn / encore happen to have some sort of watch mode too?

Reply:
It does and I can add that.